### ISSUE TITLE
Failed with Replication Experiment

### ISSUE THREAD

**Issue by jeremy7703** [qintonguav/e2e-parking-carla#26]
I tried to replicate your experiment result by generating training data myself. However, the evaluation results showed only a 1% Target Success Rate after 110 training epochs(although training loss decreased to around 0.2) and the data generation process is very unsmooth as Carla can run at only 10 fps on my pc (with intel i9-13900KF) . Could you please help by providing your training data used to get your provided model's weight for me to debug? @qintonguav @SkyrimYun @DLonng
